marklogic-client-api/src/test/java/com/marklogic/client/test/datamovement/XMLSplitterTest.java

```diff
@@ -0,0 +1,165 @@
+/*
```

Comment:
Great unit test -- covers the core functionality.

Reply:
Thanks :)